tests/drivers/build_all/spi/app.overlay

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2025 tinyvision.ai
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c0{
+	status = "okay";
+
+	sc18is606: sc18is606@28 {
+		compatible ="nxp,sc18is606";
+		status = "okay";
+		reg = <0x28>;
+
+		spi_ext: sc18is606_spi {
+			compatible = "nxp,sc18is606-spi";
+			status = "okay";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			spi-clock-frequency = <58000>;
```

Comment:
Maybe you meant to have `clock-frequency` as already defined in here?

https://github.com/zephyrproject-rtos/zephyr/blob/aa6f87dcbfeeb21cc43981a185103e062deaa1c1/dts/bindings/spi/spi-controller.yaml#L10-L14

This seems to also be a runtime config:

https://github.com/zephyrproject-rtos/zephyr/blob/aa6f87dcbfeeb21cc43981a185103e062deaa1c1/drivers/spi/spi_nrfx_spi.c#L135

https://github.com/zephyrproject-rtos/zephyr/blob/aa6f87dcbfeeb21cc43981a185103e062deaa1c1/include/zephyr/drivers/spi.h#L390-L392

Reply:
Oh wow! awesome catch, thank you!